resources/js/features/authentication/forms/login/Login.form.tsx

```diff
@@ -0,0 +1,55 @@
+import { FormConfigWithoutTransformFn } from "@felte/core";
+import { useTransContext } from "@mbarzda/solid-i18next";
+import { createMutation, useQueryClient } from "@tanstack/solid-query";
+import { FelteForm, FelteSubmit } from "components/felte-form";
+import { TextField } from "components/ui";
+import { User } from "data-access/memo-api";
+import { Component } from "solid-js";
+import { z } from "zod";
+
+export namespace LoginForm {
+  export const schema = z.object({
+    email: z.string(),
+    password: z.string(),
+  });
+
+  export const initialValues: Input = {
```

Comment:
No więc ja bym to może napisał `INITIAL_VALUES:Readonly<Input>=`, tak sobie myślę. I w sumie SCHEMA też jest runtime stałą, nie? Jak się je pisze capsem to łatwiej jest natychmiastowo zrozumieć że są stałe.

Reply:
wolę konwencję bez capsa, ale to mi się bierze z tego, że osobiście częściej piszę to jako gettery, czyli `const getInitialValues = (attr): Input => ({...})`. Taka potrzeba jest zwykle przy formularzach edcji, gdy dane początkowe są zależne od danych z zewnątrz. Możemy się umówić na formę gettera dla każdego initialValues, może też dla schema, bo czasem się mogą zdarzyć dynamiczne schemy

Reply:
No dobra, to jak uważasz.

Reply:
podmieniłem na gettery